src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java

```diff
@@ -572,7 +572,7 @@ public StructImpl download(
                     + " \"txz\", \".tar.zst\", \".tzst\", \"tar.bz2\", \".ar\", or \".deb\""
                     + " here."),
         @Param(
-            name = "stripPrefix",
+            name = "strip_prefix",
```

Comment:
This would break existing users. You could introduce a new parameter with the correct spelling and mark the old one as deprecated.